support/ebpf/util.h

```diff
@@ -0,0 +1,7 @@
+#ifndef OPTI_UTIL_H
+#define OPTI_UTIL_H
+
+#define MIN(a, b) (((a) < (b)) ? (a) : (b))
```

Comment:
I'm wondering if we should compiler builtins for this:
```suggestion
#define MIN(a, b) __builtin_elementwise_min(a,b)
```

Reply:
Doesn't compile from CGO: 

warning: implicit declaration of function ‘__builtin_elementwise_min’